modules/cnf-du-configuring-workload-partitioning.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-configuring-workload-partitioning_{context}"]
+
+= Configuring workload partitioning
+
+The following procedure outlines a high level, end to end workflow that deploys a cluster with workload partitioning enabled and pods that are correctly scheduled to run on the management CPU pool.
+
+. The user creates a machine config manifest to configure CRI-O to partition management workloads. The cpuset specified must match the reserved cpuset specified in the performance-addon-operator profile.
+
+. The user creates another machine config manifest to write a configuration file for kubelet to enable the same workload partition. The file is only readable by the kubelet.
+
+. The user runs the installer to create the standard manifests, adds their extra manifests from steps 2 and 3, then creates the cluster.
```

Comment:
Check the numbering here. I see only 2 steps above this one, should this be 1 and 2? It's referring to the crio and kubelet machine config manifest steps.

Reply:
Replaced "steps 2 and 3" > steps 1 and 2"